project_transportonly/project.json5

```diff
@@ -0,0 +1,351 @@
+            },
+            {
+                type: 'subsector',
+                class: 'Subsector',
+                name: 'DECARB 2023 Demand Subsectors',
+                display_name: 'Subsector',
+                description: 'DECARB 2023 Demand Subsectors; Vehicle Class',
+                file: 'dimensions/subsectors.csv',
+                module: 'dsgrid.dimension.standard',
+            },
+            {
+                type: 'metric',
+                class: 'EnergyEndUse',
+                name: 'DECARB 2023 End Uses',
+                display_name: 'End Use',
+                description: 'DECARB 2023 Demand End Uses; Charging Electricity Types',
+                file: 'dimensions/enduses.csv',
+                module: 'dsgrid.dimension.standard',
+            },
+            {
+                type: 'model_year',
+                class: 'ModelYear',
+                name: 'DECARB 2023 Model Years',
+                display_name: 'Model Year',
+                description: 'DECARB 2023 Model Years; Biennial 2024-2030, then every 5 years to 2050',
+                file: '../project/dimensions/model_years.csv',
+                module: 'dsgrid.dimension.standard',
+            },
+            {
+                type: 'scenario',
+                class: 'Scenario',
+                name: 'DECARB 2023 High-Resolution Demand Scenarios',
+                display_name: 'Scenario',
+                description: 'DECARB 2023 High-Resolution Demand Scenarios; DECARB High, with two charging behaviors: unmanaged and overnight slow',
+                file: 'dimensions/scenarios.csv',
+                module: 'dsgrid.dimension.standard',
+            },
+            {
+                type: 'time',
+                class: 'Time',
+                frequency: 'P0DT1H0M0.000000S',
+                name: 'Time-2018-EST-hourly-periodBeginning-noLeapDayAdjustment-total',
+                display_name: 'Time EST',
+                time_type: 'datetime',
+                leap_day_adjustment: 'none',
+                description: 'Time dimension, 2018 hourly EST, hour-beginning, no Leap Day Adjustment, total value',
+                time_interval_type: 'period_beginning',
+                str_format: '%Y-%m-%d %H:%M:%S',
+                timezone: 'EasternStandard',
+                measurement_type: 'total',
+                ranges: [
+                    {
+                        start: '2018-01-01 00:00:00',
+                        end: '2018-12-31 23:00:00',
+                    },
+                ],
+            },
+            {
+                type: 'weather_year',
+                class: 'WeatherYear',
+                name: '2018 Weather Year',
+                display_name: 'Weather 2018',
+                description: 'Weather year, 2018 only',
+                file: '../project/dimensions/weather_years.csv',
+                module: 'dsgrid.dimension.standard',
+            },
+        ],
+        subset_dimensions: [
+
+        ],
```

Comment:
nit: remove whitespace?

Reply:
Also, a bigger comment that the simplified vehicle types could be specified as Subset Dimensions instead of as a supplemental dimension.